packages/util/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@cardano-sdk/util",
+  "version": "0.2.0",
+  "description": "Observable extensions and other general utils (not cardano-specific)",
+  "engines": {
+    "node": "^14"
+  },
+  "main": "dist/index.js",
+  "repository": "https://github.com/input-output-hk/cardano-js-sdk/packages/util",
+  "author": "Mircea Hasegan <mircea.hasegan@iohk.io>",
+  "license": "MPL-2.0",
+  "scripts": {
+    "build": "tsc --build ./src",
+    "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
+    "cleanup": "shx rm -rf dist node_modules",
+    "lint": "eslint --ignore-path ../../.eslintignore \"**/*.ts\"",
+    "lint:fix": "eslint --fix --ignore-path ../../.eslintignore \"**/*.ts\"",
+    "test": "jest -c ./jest.config.js",
+    "test:e2e": "shx echo 'test:e2e' command not implemented yet",
+    "coverage": "yarn test --coverage",
+    "prepack": "yarn build",
+    "test:debug": "DEBUG=true yarn test"
+  },
+  "devDependencies": {
+    "@cardano-sdk/util-dev": " 0.2.0"
```

Comment:
This will be a circular dependency since `util-dev` depends on `core`. Let's get rid of this dependency.

I'm also not a big fan of `core` to depending on `util` as it is now, as it adds an indirect `rxjs` dependency to it :thinking: We should consider `util-rxjs` package. What do you think @rhyslbw @mirceahasegan ?

Reply:
> I'm also not a big fan of core to depending on util as it is now, as it adds an indirect rxjs dependency to it

I can't see this. Please link

Reply:
Ah core is missing a dependency in package json right now, but it's [using](https://github.com/input-output-hk/cardano-js-sdk/pull/262/files#diff-faa5b269997a2961d92093073919a379bf50ef93fb2e947bbb9d87ee7343f987R2) `util`

Reply:
> We should consider `util-rxjs` package

Yes, I agree. `util` will then have no external dependencies.

Reply:
I'm not sure what is the action item here.

Reply:
Two packages instead of one. `util` and `util-rxjs`

Reply:
I cannot find a way to do this cleanly with fixup commits that I can squash later. I will start rewriting the history of this branch to add the util-rxjs package first, then the util package. I think this is better than keeping the history where TrackerSubject was first part of util, then moved to util-rxjs.
This means that you will not see the new changes separately.

Reply:
> This means that you will not see the new changes separately.

No problem, thanks for the heads up